### ISSUE TITLE
Bug: Code section "Copied!" message is not visible for multiline code sections

### ISSUE THREAD

**Issue by OnkarRuikar** [mdn/yari#5537]
# Summary
Refer the page: https://developer.mozilla.org/en-US/docs/Web/API/Text/splitText#example
In the example part, there is a single line HTML code section and a multiline JavaScript code section.
![img](https://i.imgur.com/HXxckR1.png)

Issues: 
1. The "Copied!" message doesn't appear for the second JavaScript code section.
2. In dark mode the message isn't in inverted colors.

## Where did you spot the bug?
https://developer.mozilla.org/en-US/docs/Web/API/Text/splitText#example

## What did you expect to happen?
1. The "Copied!" message should be visible for all code sections.
2. In dark mode the messages should have inverted colors, i.e. black text on white background.

## Steps to reproduce the issue
1. Visit the page: https://developer.mozilla.org/en-US/docs/Web/API/Text/splitText#example
2. Copy the HTML code using copy icon. See the message.
3. Now copy the JavaScript code using corresponding copy icon. No message is shown.

## Possible fix:
Updating the CSS rule like this solves the issues:
```CSS
.copy-icon-message {
  /* z-index: -1000;                        <--- remove this */
  background: var(--text-primary);       /* <--- use vars to support theme change */
  color: var(--text-invert);
  ...
}
```
1. Light mode after the fix.
![fixed](https://i.imgur.com/nQDcOfn.png)
2. Dark mode after the fix.
![fixedDark](https://i.imgur.com/X0pdLiN.png)

**Comment by caugner** [comment]
Hi @OnkarRuikar, thanks for taking the time to report this! 👋 

We have addressed the dark mode issue in #5504, and this fix will be deployed by tomorrow.

As for the missing "Copied!" message, I was able to reproduce the issue:

https://user-images.githubusercontent.com/495429/157423126-25555dde-5fca-476a-894b-01bd1c22bf39.mov

**Comment by OnkarRuikar** [comment]
@caugner  done! 
Let me know if I've followed the procedure correctly.
